server/utils/comments/comment-history.util.js

```diff
@@ -0,0 +1,32 @@
+import db from '../../models';
+
+const { Comment, CommentHistory } = db;
+
+/**
+ * Insert current comment into CommentHistories table
+ * @param {int} commentId
+ * @returns {Boolean} true/false
+ */
+const saveCommentHistory = async (commentId) => {
+  let success;
+  try {
+    const currentCommit = await Comment.findByPk(commentId);
+    if (currentCommit !== null) {
+      const { comment } = currentCommit;
```

Comment:
Wouldn't it be better to use `currentComment` over `currentCommit`?

Reply:
@TheDrizzyWay Definitely. Will effect this change